sormas-ui/src/main/java/de/symeda/sormas/ui/symptoms/SymptomsForm.java

```diff
@@ -104,8 +105,9 @@
 					loc(SIGNS_AND_SYMPTOMS_HEADING_LOC) +
 					fluidRowCss(VSPACE_3,
 							//XXX #1620 fluidColumnLoc?
-							fluidColumn(8, 0, loc(SYMPTOMS_HINT_LOC))) +
+							fluidColumn(8, 0, loc(SYMPTOMS_HINT_LOC)) +
 					fluidRow(fluidColumn(6,6, locCss(CssStyles.ALIGN_RIGHT,BUTTONS_LOC)))+
+					fluidRowLocs(SYMPTOMATIC, ONSET_DATE) +
```

Comment:
Onset date and onset symptom shouldn't be split because they're related to each other. Please move onset symptom to the top as well.